examples/tutorials/sms/auto-subscribe-app/README.md

```diff
@@ -0,0 +1,59 @@
+# auto-subscribe application sample
+
+This directory contains sample related to Java SDK tutorials: [](https://developers.sinch.com/docs/sms/tutorials/sms/tutorials/java-sdk/auto-subscribe)
+
+## Requirements
+-
+- JDK 21 or later
+- [Maven](https://maven.apache.org/)
+- [ngrok](https://ngrok.com/docs)
+- [Sinch account](https://dashboard.sinch.com)
+
+## Usage
+
+### Configure application settings
+
+Application settings is using the SpringBoot configuration file: [`application.yaml`](src/main/resources/application.yaml) file and set:
+
+#### Sinch credentials
+Located in `credentials` section:
+- `key-id`: YOUR_access_key
```

Comment:
Nitpick: The object is an "access key" and this object has several properties such as the `name`, the `id`, the `secret`, the `creation date` ...
The description should reflect this in order to remove any confusion about the credentials to use: `YOUR_access_key_id` and `YOUR_access_key_secret` would be a better naming

Reply:
Good catch
Thank you / fixed